cypress/integration/own-services_spec.js

```diff
@@ -186,5 +186,33 @@ describe('Add new service provider', function () {
 
         cy.contains('Palveluntuottaja poistettiin onnistuneesti.');
     });
+});
+
+describe('Should add new associated service to Normal users "Terminaali Oy" and delete them', function () {
+    before(function() {
+        cy.login('normaluser', 'password')
```

Comment:
Does this need to define default user explicitly? 
Here test could call the custom command `cy.login()` which resolves the username pwd itself.

Reply:
Added cy.normalLogin() function to login with normaluser account.